ongs/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views import OngIdView, OngView
+
+urlpatterns = [
+    path("ongs/", OngView.as_view()),
+    path("ongs/<int:ong_id>", OngIdView.as_view()),
```

Comment:
missing `/` at the end of the url: `"ongs/<int:ong_id>/"`

Reply:
Done!